apis/management.cattle.io/v3/k8s_windows_default.go

```diff
@@ -342,21 +343,32 @@ func initWindows() {
 		if badVersions[version] {
 			continue
 		}
-		if version > maxWindowsK8sVersion {
+		if shouldIgnoreVersion(version) {
 			continue
 		}
 
 		images, ok := allK8sWindowsVersions[version]
 		if !ok {
-			panic("K8s version " + " is not found in AllK8sWindowsVersions map")
+			panic("K8s version " + version + " is not found in AllK8sWindowsVersions map")
 		}
 
 		K8sVersionWindowsSystemImages[version] = images
 	}
+}
 
-	if maxWindowsK8sVersion > DefaultK8s {
-		if _, ok := K8sVersionWindowsSystemImages[DefaultK8s]; !ok {
-			panic("Default K8s version " + DefaultK8s + " is not found in k8sVersionsCurrent list")
+func shouldIgnoreVersion(version string) bool {
+	// we stopped building Windows images at 1.13.9, so we ignore checking anything above 1.13.5 (which was the last release before 1.13.9)
+	versionParts := strings.Split(version, ".")
+	minor := versionParts[1]
+	if minor == "13" {
```

Comment:
what happens to the checks for the other k8s minor versions? 

what if we update k8s 1.12 or below versions? (unlikely)